doc/getting-started/acrn-sanity-test.rst

```diff
@@ -0,0 +1,352 @@
+ACRN Sanity Test
```

Comment:
Are these test cases described below valid for *all* HW platforms we support? If not, let's call out which are expected to work as described.